docs/includes/hot-backup-elasticsearch-snapshots.md

```diff
@@ -0,0 +1,91 @@
+Before creating Elasticsearch snapshots, ensure Elasticsearch has the appropriate permissions to write to the snapshot repository.
+
+For shared file systems:
+
+!!! Example ""
+
+    ```bash
+    chown -R elasticsearch:elasticsearch </mnt/backup>
+    chmod -R 770 </mnt/backup>
+    ```
```

Comment:
```suggestion
    ```bash
    chown elasticsearch:elasticsearch </mnt/backup>
    chmod 770 </mnt/backup>
    ```
```
Since all backups will be under `/mnt/backup` (and not only ElasticSearch), we should be careful about assigning ownership and permissions recursively.